app/uploaders/supporting_evidence_attachment_uploader.rb

```diff
@@ -2,6 +2,9 @@
 
 # Files uploaded as part of a form526 submission that will be sent to EVSS upon form submission.
 class SupportingEvidenceAttachmentUploader < EVSSClaimDocumentUploaderBase
+  before :store, :log_transaction_start
+  after :store, :log_transaction_complete
```

Comment:
This could easily be moved up the inheritance chain for visibility across all EVSSClaimDocumentUploader actions, however for this work we are scoping it to our usecase.